SdsClient.js

```diff
@@ -64,31 +66,30 @@ module.exports = {
 
     // returns an access token
     this.getToken = function (clientId, clientSecret, resource) {
-      return restCall({
+      return axios({
         url: resource + '/identity/.well-known/openid-configuration',
         method: 'GET',
         headers: {
           Accept: 'application/json',
+          'Accept-Encoding': 'gzip',
         },
-        gzip: true,
       })
         .then(function (res) {
-          var obj = JSON.parse(res);
+          var obj = res.data;
           authority = obj.token_endpoint;
 
-          return restCall({
-            url: authority,
-            method: 'POST',
+          var body = new URLSearchParams({
+            grant_type: 'client_credentials',
+            client_id: clientId,
+            client_secret: clientSecret,
+          });
+
+          console.log(body.toString());
```

Comment:
This looks like it should've only been there for debugging. Is this intended to be part of the sample?

Reply:
fixed typo